src/Helpers.js

```diff
@@ -3,6 +3,7 @@ import { InjectedConnector, UserRejectedRequestError as UserRejectedRequestError
 import { WalletConnectConnector, UserRejectedRequestError as UserRejectedRequestErrorWalletConnect } from '@web3-react/walletconnect-connector'
 import { toast } from 'react-toastify'
 import { useWeb3React, UnsupportedChainIdError } from '@web3-react/core'
+import { getDefaultProvider } from "@ethersproject/providers";
```

Comment:
there is `ethers` imported already, i guess `ethers.getDefaultProvider` is the same thing?